src/locale/states/za.json

```
@@ -0,0 +1,11 @@
+{
+    "EC": "Eastern Cape",
+    "FS": "Free State",
+    "GP": "Gauteng",
+    "KZN": "Kwazulu-Natal",
+    "LP": "Limpopo",
+    "MP": "Mpumalanga",
+    "NC": "Northern Cape",
+    "NW": "North-West",
```

According to https://www.gov.za/provinces, this is `North West`.  Should this be updated to match?

```suggestion
    "NW": "North West",
```

---

That would depend on whether we are going to be using ISO 3166 data in future, that's where I got my string from. Technically in formal English North-West is hyphenated.

I don't mind either way, so it's a matter of what's going to be less of a maintenance headache going forward.